Makefile

```diff
@@ -0,0 +1,61 @@
+# Variables
+REPO_FOLDER = charmed-superset-rock
+REPO = https://github.com/canonical/$(REPO_FOLDER).git
+ROCK_DEV = rock-dev
+ROCK_VERSION = 2.1.0
+UBUNTU_VER = 22.04
+DOCKER_NAME = superset-ui-services
+DOCKER_PORT = 8088
+DOCKER_ARGS = superset-ui -g 'daemon off:' \; start superset-ui
+
+# Targets
```

Comment:
What happens if I run `make all` twice, or `make clean` twice? I don't know how smart `make` is. Does it run PHONY targets every time? How would `multipass` or `snap` handle duplicate commands?

Reply:
you make a good point (pun intended!). I wil work on making the targets idempotent